### DISCUSSION TITLE
KRATOS_URL

### DISCUSSION THREAD

**Discussion by elenezet** [Security-Onion-Solutions/securityonion#7420] [General] [answered]
Good morning all
I have a problem with Kratos when I use certain commands. Here is an example :
 sudo so-user-list 
curl: (18) transfer closed with outstanding read data remaining
Unable to communicate with Kratos; specify URL via KRATOS_URL environment variable

Do you know where to set this variable?

**Comment by dougburks** [comment, marked as answer]
Looks like that error message comes from https://github.com/Security-Onion-Solutions/securityonion/blob/master/salt/common/tools/sbin/so-user#L86.

Seems like maybe Kratos isn't running.

What is the output of the following?
```
sudo so-status
```

**Reply by elenezet** [reply]
Hello Doug,
Everything was OK. I removed the proxy export in /etc/environment and everything seems OK now.
